src/auth/auth.module.ts

```diff
@@ -0,0 +1,27 @@
+import { Module } from '@nestjs/common';
+import { JwtModule, JwtService } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+import { AuthController } from './auth.controller';
+import { RedisModule } from '@liaoliaots/nestjs-redis';
+import { Env } from 'src/env';
+import { JwtStrategy } from './jwt.strategy';
+import { GithubModule } from 'src/github/github.module';
+
+@Module({
+  imports: [
+    GithubModule,
+    PassportModule,
+    JwtModule.register({
+      secret: Env.JWT.AUTH_TOKEN,
+      signOptions: { expiresIn: Env.JWT.EXPIRES_IN },
+    }),
+    RedisModule.forRoot({
+      config: { host: Env.REDIS.HOST, port: Env.REDIS.PORT },
```

Comment:
PullProofAI

The import statements in the `AuthModule` are not organized properly. It's recommended to group imports by their source, with third-party imports coming before local imports.

```diff
6c6
< import { RedisModule } from '@liaoliaots/nestjs-redis';


<!-- This is an auto-generated comment by PullProofAI -->